docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -14,7 +14,7 @@ Using the command line Roslyn can be developed using the following pattern:
 1. [Visual Studio 2017 Update 3](https://www.visualstudio.com/vs/)
     - Ensure C#, VB, MSBuild and Visual Studio Extensibility are included in the selected work loads
     - Ensure Visual Studio is on Version "15.3" or greater
-1. [.NET Core SDK 2.0](https://www.microsoft.com/net/download/core)
+1. [.NET Core SDK 2.1](https://www.microsoft.com/net/download/core) (here are direct links while the document is getting updated: [Windows x64 installer](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.1-preview-007094/dotnet-sdk-2.1.1-preview-007094-win-x64.exe), [Windows x86 installer](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.1-preview-007094/dotnet-sdk-2.1.1-preview-007094-win-x86.exe))
```

Comment:
Are these anticipated to be the final build? Should we rephrase the parenthetical to say "if you don't see them, the current previews are...)